client/landing/gutenboarding/variables.scss

```
@@ -1,4 +1,6 @@
 // Reusable style variables
 $gutenboarding-header-height: 64px;
-$gutenboarding-page-content-padding-desktop: 60px;
-$gutenboarding-page-content-padding-mobile: 44px;
+$gutenboarding-heading-padding-top-desktop: 64px;
+$gutenboarding-heading-padding-bottom-desktop: 80px;
+$gutenboarding-heading-padding-top-mobile: 44px;
+$gutenboarding-heading-padding-bottom-mobile: 50px;
```

Maybe merge every couple into one?
```suggestion
$gutenboarding-heading-padding-desktop: 64px 0 80px;
$gutenboarding-heading-padding-mobile: 44px 0 50px;
```

---

I was thinking we might need the values independently for some alignment but you're right. We can separate them when needed 👍